zuul.d/projects.yaml

```diff
@@ -10,6 +10,12 @@
     name: github.com/ansible-collection-migration/ansible-base
     default-branch: devel
 
+- project:
+    name: github.com/ansible-collections/amazon.aws
+    default-branch: master
+    templates:
```

Comment:
This isn't correct, we now want all projects to atleast have system-required. This is because we add the github-workflows jobs now.  Noop isn't needed, so we can skip this